### PR TITLE
Add 404 page

### PR DESCRIPTION
**Pull request by taskinoz**
Taken from the old master server.

@pg9182 you'll need to add an entry in nginx or atlas to use this page as a 404.

### CODE REVIEW

**Review by pg9182 — CHANGES_REQUESTED** [review]
~~Can you rename it to `.errors/404.html`?~~

Edit: Nevermind, I'm going to support error pages in the root directory since it's what GitHub Pages supports, and we're eventually going to be using it.

**Review by pg9182 — CHANGES_REQUESTED** [review]
Can you move the error page to `404.html` in the root dir?

**Review by pg9182 — CHANGES_REQUESTED** [review]

File: dist/404.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>404 - Titan Not Found</title>
+    <link rel="stylesheet" href="/style/index.css">
+    <link rel="stylesheet" href="/style/common.css">
+    <link rel="stylesheet" href="/style/404.css">
+</head>
+<body>
+    <div id="videofull">
+        <video class="lazy" autoplay loop muted playsinline poster="assets/NorthstarPromoPoster.jpg">
+            <source src="assets/NorthstarPromoVid.mp4">
```
[the remark this answers is not in the document]
Comment (taskinoz) [reply dist/404.html:15]: What do you mean by this

File: dist/404.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <div id="centerlogo">
+            <h1>404</h1>
+            <h2>Titan Not Found</h2>
+        </div>
+        <div id="mainbuttons">
+            <a href="/" class="button">Go Home</a>
+        </div>     
+    </div>
+</body>
+</html>
```
[the remark this answers is not in the document]
Comment (taskinoz) [reply dist/404.html]: Added

File: dist/style/404.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    width: max-content;
+    user-select: none !important;
+    */
+}
+
+a.button {
+    margin: 8px;
+    padding: 16px 32px;
+    height: 50px;
+}
```
[the remark this answers is not in the document]
Comment (taskinoz) [reply dist/style/404.css]: Added

File: dist/style/404.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+#mainbuttons {
+    margin-top: 0;
+    top: calc((30% + 256px) - 72px);
+    left: 50%;
+    transform: translateX(-50%);
+    width: max-content;
+    user-select: none !important;
+    */
```
[the remark this answers is not in the document]
Comment (taskinoz) [reply dist/style/404.css]: Removed

File: dist/404.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>404 - Titan Not Found</title>
+    <link rel="stylesheet" href="/style/index.css">
+    <link rel="stylesheet" href="/style/common.css">
+    <link rel="stylesheet" href="/style/404.css">
+</head>
+<body>
+    <div id="videofull">
+        <video class="lazy" autoplay loop muted playsinline poster="assets/NorthstarPromoPoster.jpg">
```
Comment (pg9182) [inline dist/404.html:14]: This too.

Comment (taskinoz) [reply dist/404.html]: Updated

**Review by pg9182 — APPROVED** [review]
LGTM

**Pull request merged**